.github/workflows/evaluate-dependency-health.yml

```
@@ -0,0 +1,43 @@
+name: Evaluate Dependency Health Score
+on:
+  pull_request:
```

```suggestion
  pull_request_target:
```

`pull_request` run in the forked repo and can leak secrets. `pull_request_target` will run in the Elastic repo.